applicationLibrary/molecularDynamics/molecularDynamicsLibrary/LJFunctor.h

```diff
@@ -354,8 +353,6 @@ class LJFunctor
       _aosThreadDataFLOPs[threadnum].numGlobalCalcsN3 += numGlobalCalcsSum;  // Always N3 in Single SoAFunctor
     }
     if (calculateGlobals) {
-      const int threadnum = autopas::autopas_get_thread_num();
```

Comment:
This was redundant as it shadows the same variable defined at the start of the function.